src/storage/src/client/upload_object_buffered.rs

```diff
@@ -797,7 +800,7 @@ async fn next_chunk<T>(
     payload: &mut InsertPayload<T>,
     remainder: Option<bytes::Bytes>,
     target_size: usize,
-) -> Result<(VecDeque<bytes::Bytes>, Option<bytes::Bytes>)>
+) -> Result<(VecDeque<bytes::Bytes>, usize, Option<bytes::Bytes>)>
```

Comment:
optional: A comment describing the return value could be helpful when I forget the interface, which should happen by next week.

Reply:
Fixed by using a struct